docs/Onboarding.md

```diff
@@ -24,15 +24,32 @@ By default, PostgreSQL typically has a `postgres` database.
 You can use `psql` in the CLI to create a database by running this SQL statement: `CREATE DATABASE nerpm;`.
 Naming the new database `nerpm` will ensure it matches with the database URL specified in the project preparation section below.
 
-### Project Preparation
+### ENV Setup
 
 Add a `.env` file to the project root directory via `touch .env` in the CLI or creating a file in your IDE.
 Paste the following line into the `.env` file and replace `<USERNAME>` with your computer username.
 `DATABASE_URL="postgresql://<USERNAME>:@localhost:5432/nerpm?schema=public"`
 
+### Initial Database Migration
+
+In order to run the database for the first time, you will need to execute the following commands in the CLI.
+Run `npm prisma:migrate`.
+You will need to also input a name for the migration in the `name` field that is prompted.
+Then run `npm prisma:seed`.
```

Comment:
why not just have them run `prisma:reset` instead of these two commands? Would it work? Is there a difference?